### PR TITLE
HWKAPM-643 Display instance diagram for simple spans

### PR DESCRIPTION
**Pull request by objectiser**

### CODE REVIEW

**Comment by objectiser** [comment]
@pavolloffay Could you review please?

Reviewer note:
* Problem found when starting up APM server before doing 'docker-compose up' on the polyglot example, which resulted in some db activity being reported
* These activities display in the DT view, and the trace instances are displayed in the table, but selecting the instance diagram did not work

File: server/elasticsearch/src/main/java/org/hawkular/apm/server/elasticsearch/SpanServiceElasticsearch.java [hunk 1]
```diff
@@ -246,7 +246,7 @@ public Trace getTraceFragment(String tenantId, String id) {
     @Override
     public Trace getTrace(String tenantId, String id) {
         Span span = getSpan(tenantId, id);
-        if (span != null && span.serverSpan()) {
+        if (span == null || (span != null && span.serverSpan())) {
```
Comment (pavolloffay) [inline server/elasticsearch/src/main/java/org/hawkular/apm/server/elasticsearch/SpanServiceElasticsearch.java]: `(span == null || (span != null && span.serverSpan()))` is the same as `(span == null || span.serverSpan())`

Comment (objectiser) [inline server/elasticsearch/src/main/java/org/hawkular/apm/server/elasticsearch/SpanServiceElasticsearch.java]: yes :)

**Comment by objectiser** [comment]
@pavolloffay Thanks - I'll merge after CI has finished.

**Pull request merged**